grunt/index.d.ts

```diff
@@ -796,8 +796,8 @@ declare namespace grunt {
              *
              * @note taskFunction.apply(scope: grunt.task.IMultiTask<any>, args: any[])
              */
-            registerMultiTask(taskName: string, taskFunction: Function): void
-            registerMultiTask(taskName: string, taskDescription: string, taskFunction: Function): void
+            registerMultiTask(taskName: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
+            registerMultiTask(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<any>, ...args: any[]) => void): void;
```

Comment:
I guess I was expecting you would go the other way with adding the type parameter, but this works for me. I might just have trouble understanding the ramifications since it has been a while since I have used TypeScript. Would this not work for most use cases?

```
registerMultiTask<T>(taskName: string, taskDescription: string, taskFunction: (this: IMultiTask<T>, ...args: any[]) => void): void
```

If not, could you explain why? I'm okay with using `any`, but it seems like allowing the caller to constrict the type could be useful.